### ISSUE TITLE
Variant discovery from a pan-genome reference

### ISSUE THREAD

**Issue by Kmanjor** [vgteam/vg#3670]
Hi,

Can vg call SNP or Indel in insertion? like this figure. They used linear pan-genome reference with the insertion included, and they found an SNP present in the insertion.

Thanks. 

Reference figure5: Sherman RM, Salzberg SL. Pan-genomics in the human genome era. Nat Rev Genet. 2020 Apr;21(4):243-254. doi: 10.1038/s41576-020-0210-7.
![image](https://user-images.githubusercontent.com/42490165/170219885-382e6030-1de5-45ea-9a88-dcc2a2de56f8.png)

**Comment by Kmanjor** [comment]
I know vg can augment novel variants in the graph from the gam alignment file. But I don't know whether vg can augment variants in insertion. Thank you~

**Comment by faithokamoto** [comment]
Yes, see https://github.com/vgteam/vg/wiki/VCF-export-with-vg-deconstruct#nested-decomposition-experimental
